synapse/swarm/SYNTAX.md

```diff
@@ -0,0 +1,100 @@
+## Lift
+  Supported comparisons: Basic
+
+Lift all foo tufos.
+
+    foo
+
+Lift all foo tufos from cortexes tagged my.core
+
+    my.core/foo
+
+Lift all foo tufos with a foo:bar prop with the given value.
+
+    foo:bar=10
+    foo:bar="ten"
+
+## Must/Cant
+Supported comparisons: Basic Extended
+
+Remove all tufos from our data set that do not have a foo:baz prop.
+
+    ... +foo:baz
+
+Remove all tufos from our data set that have a foo:baz prop.
+
+    ... -foo:baz
+
+Remove all tufos from our data set that do not have a foo:baz prop equal to the given value
+
+    ... +foo:baz=10
+    ... +foo:baz="ten"
+
+Remove all tufos from our data set that have a foo:baz prop equal to the given value
+
+    ... -foo:baz=10
+    ... -foo:baz="ten"
+
+## Join
+Supported comparisons: Basic
+
+Join all foo tufos whose foo:bar prop matches a baz:faz prop in our data set.
+
+    ... <foo:bar=baz:faz
```

Comment:
not a huge fan of < as join macro due to being confused with less than. liked & but it's now in use for logic... thoughts?